File: taobao-user-interface/src/main/java/com/taobao/user/service/CommentService.java
package com.taobao.user.service;

import com.taobao.pojo.TbComment;
import com.taobao.pojo.group.Result;

import java.util.List;

public interface CommentService {

    /**
     * 通过评论等级得到评论
     */
    public List<TbComment> findByStar(Long gooodsId,Integer star);

    /**
     * 得到商品的所有评价
     *
     */
    public List<TbComment> findByGoodsId(Long goodsId);

    /**
     * 创建一条评价
     * @param comment
     * @return
     */
    public Result addComment(TbComment comment);
}

File: taobao-user-web/src/main/webapp/js/service/commentService.js
app.service('commentService',function ($http) {
    
    //得到商品的所有评论
    this.findByStar = function (goodsId,star) {
        return $http.get("comment/findByStar.do?goodsId="+goodsId+"&star="+star);
    }

    //添加一条评论
    this.addComment = function (entity) {
        return $http.post("comment/addComment.do",entity);
    }
});